### PR TITLE
FF112 Relnote: Remove IDBMutableFile and friends

### PR DESCRIPTION
**Pull request by hamishwillee**
FF112 permanently removes `IDBMutableFile`, `IDBFileRequest`, `IDBFileHandle`, `IDBDatabase.createMutableFile()` from FF in https://bugzilla.mozilla.org/show_bug.cgi?id=1500343

This adds a release note.

Related docs work can be tracked in #25360

### CODE REVIEW

**Comment by hamishwillee** [comment]
Hi @dipikabh 
Since Brian is away, can you please review this?

File: files/en-us/mozilla/firefox/releases/112/index.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ This article provides information about the changes in Firefox 112 that affect d
 
 ### APIs
 
+
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/112/index.md:40]:
```suggestion
```

**Comment by hamishwillee** [comment]
@dipikabh Could you please review this.

**Comment by hamishwillee** [comment]
Or @teoli2003

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**